praw/exceptions.py

```diff
@@ -89,7 +92,14 @@ def __init__(self, items: List[Union[RedditErrorItem, List[str]]]):
         super().__init__(*self.items)
 
 
-APIException = RedditAPIException
+def APIException(*exception_args, **exception_kwargs):
+    """Warn of deprecation and return aliased exception."""
+    warn(
```

Comment:
This isn't quite what I meant. I meant to deprecate accessing any of the attributes through RedditAPIException.